truffle-config.js

```diff
@@ -16,6 +16,24 @@ module.exports = {
   //    port: 7545,
   //    network_id: "*"
   //  }
-  //}
+  //},
+
   //
+  db: {
+    // Truffle-DB is disabled by default. 
+    enabled: false,
+    host: "127.0.0.1",
+
+    // You can also use a local db storage adapter by uncommenting this section
+    //
+    adapter: {
+      name: "sqlite",
+      settings: {
+        // You may also choose a different directory name
+        // Note. it will always be relative to the Project root
```

Comment:
```suggestion
        // Note: the directory name will always be relative to the Project root
```

Reply:
It won't always be relative to the project root - it might be an absolute path

Reply:
oh, good to know!